doc/classes/Window.xml

```diff
@@ -558,8 +558,9 @@
 		<member name="always_on_top" type="bool" setter="set_flag" getter="get_flag" default="false">
 			If [code]true[/code], the window will be on top of all other windows. Does not work if [member transient] is enabled.
 		</member>
-		<member name="auto_translate" type="bool" setter="set_auto_translate" getter="is_auto_translating" default="true">
+		<member name="auto_translate" type="bool" setter="set_auto_translate" getter="is_auto_translating" default="true" is_deprecated="true">
```

Comment:
The checks are complaining that `is_deprecated="true"` should be `deprecated=""`, but no other doc file has it like that.

Reply:
See #81458.